cmake/platforms/xlnx-a53-freertos.cmake

```diff
@@ -1,5 +1,5 @@
 set (CMAKE_SYSTEM_PROCESSOR "aarch64"           CACHE STRING "")
-set (MACHINE                "zynqmp_a53"        CACHE STRING "")
```

Comment:
keep the alignement

Reply:
After addressing this comment it's good to go.

Reply:
but still not aligned